### PR TITLE
Add cutlass submodule for sgl-kernel

### PR DESCRIPTION
**Pull request by ispobock**
## Motivation

Include CUTLASS (v3.6.0) for kernels dev.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "sgl-kernel/3rdparty/cutlass"]
```
Comment (zhyncs) [inline .gitmodules:1]: May we specify the latest tag v3.6.0

**Comment by zhyncs** [comment]
cc @merrymercy

**Review by zhyncs — APPROVED** [review]

**Review by merrymercy — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (ispobock) [reply .gitmodules:1]: It seems tag cannot be specified. But I have specified the commit.

Comment (zhyncs) [reply .gitmodules:1]: I see

**Pull request merged**